docs/guides/about/glossary.md

```diff
@@ -791,6 +811,12 @@ NetCDF subsetting, OGC WCS and WMS.
 
 For more information, see [NCI: Data ](https://nci.org.au/our-services/data-services).
 
+{#tile}
+## Tile
+
+[Scenes](#scene) are resampled onto a grid that is fixed to the Earth's coordinates, measured in units of kilometers aligned with the North Pole.
+This resampled grid is made of 100 &times; 100 km regions called tiles.
```

Comment:
This is a tricky one: tiles are just a way of dividing our data into smaller chunks for analysis and delivery - they can come in all sorts of shapes and sizes. The one mentioned here is our older Collection 2 grids (no longer used), and we no longer resample most of our scene-based data into grids at all.

I'd either remove this one, or make it more generic - maybe


> Tiles are a way of dividing data products into smaller, more manageable regions for analysis and data access. Each tile represents a specific geographic area, and multiple tiles can be combined into a grid to cover larger regions. In DEA, summary products like DEA GeoMAD are provided using [96 x 96 km DEA Summary Products grid tiles](https://knowledge.dea.ga.gov.au/guides/reference/collection_3_summary_grid/).

Reply:
I like your wording. I've now got this:

Tiles are a way of dividing data products into smaller pieces to enable the data to be handled and analysed more easily. 
Each tile represents a specific geographic area and multiple tiles can be combined into a grid to cover larger regions.
For instance, the [DEA GeoMAD][GeoMad] product is provided using 96 &times; 96 km grid tiles, characteristic of DEA Summary Products.

[GeoMad]: https://knowledge.dea.ga.gov.au/data/product/dea-geometric-median-and-median-absolute-deviation-landsat/